libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -1780,6 +1804,32 @@ def write_AIRSPEED_config(self, f):
         if len(devlist) > 0:
             f.write('#define HAL_AIRSPEED_PROBE_LIST %s\n\n' % ';'.join(devlist))
 
+    def write_DATAFLASH_config(self, f):
+        '''write dataflash config defines'''
+        # DATAFLASH block|littlefs:<w25nxx>
```

Comment:
Do you plan to add the other flash types for block logging and migrate everything over fully before merging? Or keep the old defines for the NOR flashes?

Reply:
Plan to keep the old defines for now, but eventually I would like to move stuff over.